blocks/newslist/newslist.js

```diff
@@ -490,7 +490,7 @@ export default async function decorate(block) {
           <span class="newslist-filter-arrow"></span>
         </label>
         <input type="text" id="newslist-filter-input" title="${pDateRange}" name="date" value="" size="40" placeholder="${pDateRange}" maxlength="60" disabled>
-        <input type="submit" value="" disabled>
+        <input type="submit" value="" aria-label="calendar" disabled>
```

Comment:
but the actual widget is the input (text) above right?
should we put the label on the submit?

Reply:
![Screenshot 2023-11-03 at 5 35 31 PM](https://github.com/hlxsites/accenture-newsroom/assets/7074782/7fb277a2-c21b-4a05-b834-98f7fae48ff5)

PSI complaining about this.